### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by varshit97**
@erilyth I added the people who worked on this project to the readme file.
Please check the commit

Thanks
Varshit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,9 @@ Instructions:
 * Reach the green marker to finish the game.
 * Avoiding colliding with any boxes throughout the game.
 * Click/Touch to change the camera. There are 2 cameras, a follow cam and a side view cam.
+
+People who worked on this
+*Batchu Venkat Vishal [Team Leader]
```
Comment (erilyth) [inline README.md]: Please add spaces after the "*", to fit with the style.

**Comment by varshit97** [comment]
@erilyth See the new commit. Anything else to be changed?

**Comment by erilyth** [comment]
Merged.

**Comment by varshit97** [comment]
@erilyth Thanks for making me a part of your open source project. Looking forward to making more changes :p

**Pull request merged**